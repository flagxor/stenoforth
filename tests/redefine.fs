: star 44 emit ;
star
: test 10 for star next ;
test
: star 43 emit ;
test
redefine star
test
: asterix 42 emit ;
test
redefine star
test
bye
