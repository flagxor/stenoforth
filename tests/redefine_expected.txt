StenoForth 1.0
0 0 0 0 ok> : star 44 emit ;
0 0 0 0 ok> star,
0 0 0 0 ok> : test 10 for star next ;
0 0 0 0 ok> test,,,,,,,,,,,
0 0 0 0 ok> : star 43 emit ;
0 0 0 0 ok> test,,,,,,,,,,,
0 0 0 0 ok> redefine star
0 0 0 0 ok> test+++++++++++
0 0 0 0 ok> : asterix 42 emit ;
0 0 0 0 ok> test+++++++++++
0 0 0 0 ok> redefine star
0 0 0 0 ok> test***********
0 0 0 0 ok> bye
